EXPECTATION_FAILED
__future_module__
InvalidURL
REQUEST_TIMEOUT
NOT_IMPLEMENTED
PROXY_AUTHENTICATION_REQUIRED
FOUND
NOT_FOUND
BAD_GATEWAY
UnknownProtocol
LOCKED
UnknownTransferEncoding
StringIO
ACCEPTED
IM_USED
HTTP
responses
HTTPResponse
urlsplit
UnimplementedFileMode
PAYMENT_REQUIRED
RESET_CONTENT
PARTIAL_CONTENT
warnings
LENGTH_REQUIRED
REQUEST_URI_TOO_LONG
MULTI_STATUS
socket
UPGRADE_REQUIRED
HTTPMessage
FORBIDDEN
re
CREATED
NotConnected
INSUFFICIENT_STORAGE
array
NON_AUTHORITATIVE_INFORMATION
NOT_EXTENDED
HTTPSConnection
py3kwarning
REQUEST_ENTITY_TOO_LARGE
SWITCHING_PROTOCOLS
CONTINUE
HTTPException
CannotSendRequest
FAILED_DEPENDENCY
CannotSendHeader
UNSUPPORTED_MEDIA_TYPE
HTTPS
LineAndFileWrapper
HTTP_VERSION_NOT_SUPPORTED
NO_CONTENT
OK
INTERNAL_SERVER_ERROR
error
NOT_MODIFIED
FakeSocket
PRECONDITION_FAILED
SEE_OTHER
UNPROCESSABLE_ENTITY
HTTPS_PORT
MOVED_PERMANENTLY
IncompleteRead
MULTIPLE_CHOICES
GONE
BadStatusLine
NOT_ACCEPTABLE
HTTP_PORT
CONFLICT
ssl
True
USE_PROXY
METHOD_NOT_ALLOWED
BAD_REQUEST
os
LineTooLong
PROCESSING
TEMPORARY_REDIRECT
UNAUTHORIZED
ImproperConnectionState
REQUESTED_RANGE_NOT_SATISFIABLE
GATEWAY_TIMEOUT
MAXAMOUNT
HTTPConnection
PY3
mimetools
ResponseNotReady
SERVICE_UNAVAILABLE
